src/node.cc

```diff
@@ -4910,6 +4943,8 @@ int Deinitialize() {
 
   deinitialize::_DeinitV8();
 
+  // TODO(js): Do we need to tear down OpenSsl?
```

Comment:
I am by no means an expert on OpenSSL, but according to [their wiki](https://wiki.openssl.org/index.php/Library_Initialization#Cleanup), there are some cleanup mechanisms we should look into.
Unfortunately I don't know how node uses OpenSSL, so I have no idea which parts of the cleanup from the wiki would be necessary, but I think we should investigate this to avoid any memory leaks we can (we will have to live with [some introduced by OpenSSL](https://rt.openssl.org/Ticket/Display.html?id=2561&user=guest&pass=guest)). 
Feel free to place this into a new issue, as I think it'd break the size of the issue this PR is dealing with.

Reply:
I created an issue for this: #71